package.json

```diff
@@ -25,12 +25,14 @@
   "dependencies": {
     "cors": "^2.8.5",
     "express": "^4.17.3",
+    "lodash": "^4.17.21",
```

Comment:
is it possible to just add lodash to devDependencies, since we're only using it in unit tests right now?